mission/functions/core/teams/fn_force_team_change.sqf

```diff
@@ -53,6 +52,7 @@ _nextPlayerTeamArray pushBackUnique _player;
 	[] call vn_mf_fnc_task_refresh_tasks_client;
 	[] call vn_mf_fnc_tr_overview_team_update;
 	[] call vn_mf_fnc_update_channels;
+	[] call vn_mf_fnc_apply_unit_traits;
+	[] call vn_mf_fnc_action_trait;
```

Comment:
`vn_mf_fnc_apply_unit_traits` needs to be called to set the new traits on the player BEFORE we readd the wheel menu with the call to `vn_mf_fnc_action_trait`